avm/res/api-management/service/api/main.bicep

```diff
@@ -99,12 +105,12 @@ param value string?
 @description('Optional. Criteria to limit import of WSDL to a subset of the document.')
 param wsdlSelector object?
 
-resource service 'Microsoft.ApiManagement/service@2021-08-01' existing = {
+resource service 'Microsoft.ApiManagement/service@2023-05-01-preview' existing = {
   name: apiManagementServiceName
 }
 
-resource api 'Microsoft.ApiManagement/service/apis@2021-08-01' = {
-  name: name
+resource api 'Microsoft.ApiManagement/service/apis@2022-08-01' = {
+  name: apiName
```

Comment:
Should be `name` as per spec [Inputs - Parameter/Variable Naming](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-rmfr6---category-inputs---parametervariable-naming)